recipe/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head> 
+      <title>My Chocoflan Recipe</title>
+      <link rel="stylesheet" type="text/css" href="styles.css" /></link>
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Monoton&display=swap" rel="stylesheet">
+
+  </head>
+    <body>
+
+        <div class = "Top">
+            <h1>Chocoflan</h1>
+            <p class = "description">This is the famous Chocoflan that has been passed down through history. My mom has made this recipe since she was 16 right before her mom passed away. It wasn't until I was 16 that I was introduced to this recipe.</p> 
+        </div>
+           
+        <div class = "left">
+              <div>
+             <img id = "img" src="https://www.mylatinatable.com/wp-content/uploads/2019/02/chocoflan-5-1024x716.jpg.webp" alt="">
```

Comment:
since ID tags need to be unique in an `html` file, I'd recommend that you use a ID here that is a bit more descriptive than `img` since it's likely that this won't be the only image in this entire file